workspaces/controller/internal/controller/workspace_controller.go

```diff
@@ -906,7 +902,15 @@ func generateService(workspace *kubefloworgv1beta1.Workspace, imageConfigSpec ku
 // generateWorkspaceStatus generates a WorkspaceStatus for a Workspace
 func generateWorkspaceStatus(workspace *kubefloworgv1beta1.Workspace, pod *corev1.Pod) kubefloworgv1beta1.WorkspaceStatus {
 	status := workspace.Status
-
+	if workspace.Spec.Paused != nil && *workspace.Spec.Paused {
```

Comment:
Lets sepperate this out with a comment that explains we set the pauseTime to 0 when the workspace is not paused.

```suggestion

        // if workspace is paused, update the `status.pauseTime`
        // NOTE: when the workspace is not paused, the pauseTime should be 0
	if workspace.Spec.Paused != nil && *workspace.Spec.Paused {
```